include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -2143,8 +2143,8 @@ namespace internal
   // parallel::distributed::Vector is really the same as stored in MatrixFree
   template <typename VectorType>
   inline
-  void check_vector_compatibility (const VectorType                             &vec,
-                                   const internal::MatrixFreeFunctions::DoFInfo &dof_info)
+  void check_vector_compatibility (const VectorType                             &NDEBUG_UNUSED_PARAMETER(vec),
+                                   const internal::MatrixFreeFunctions::DoFInfo &NDEBUG_UNUSED_PARAMETER(dof_info))
   {
```

Comment:
We have the habit of simply void-casting the arguments to silence warnings about unused parameters:
```
(void) vec;
(void) dof_info;
```
Can you simply do that?

Otherwise we would have to update the other 506 places as well.